plugins/SLASHED-for-WP/includes/class-inventory.php

```
@@ -0,0 +1,807 @@
+	public static function get() {
+		if ( isset( self::$cache[ static::class ] ) ) {
+			return self::$cache[ static::class ];
+		}
+
+		// Prime the cache with the resolved inventory BEFORE applying the
+		// filter. A filter callback is allowed to query inventory data via
+		// the public Slashed_Inventory::get_*() helpers; without a
+		// pre-primed cache that re-entry would call resolve() recursively
+		// and never terminate. With the cache primed, recursive get() calls
+		// short-circuit on the first line above.
+		self::$cache[ static::class ] = self::sanitize_inventory( static::resolve() );
```

PHPStan (2.2.1)

[error] Unsafe call to private method Slashed_Inventory::resolve() through static::.


(staticClassAccess.privateMethod)